src/js/components/recipient/topFive/TopFiveSection.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * TopFiveSection.jsx
+ * Created by Kwadwo Opoku-Debrah 07/10/18
+ */
+
+import React from 'react';
+
+import { recipientCategories as topCategories } from 'dataMapping/recipients/topCategories';
+import TopFiveContainer from 'containers/recipient/topFive/TopFiveContainer';
+
+export default class TopFiveSection extends React.Component {
+    render() {
+        const content = topCategories.map((category) => (
+            <TopFiveContainer
+                key={category}
+                category={category} />
+        ));
+
+        return (
+            <div
+                className="recipient-section topfive"
+                id="recipient-top-five">
+                <h3 className="state-section__title">
+                    Top 5
+                </h3>
+                <hr className="results-divider" />
+                <div className="state-section__description">
```

Comment:
Double check with the designers, but I believe we'll want to remove the last sentence for now since the links are out of scope.